2024-12-08T18:40:21,717 Using pip 24.3.1 from C:\Users\ASUS\envhopital\Lib\site-packages\pip (python 3.12)
2024-12-08T18:40:21,717 Non-user install because user site-packages disabled
2024-12-08T18:40:21,954 Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-build-tracker-uzy8h0iy
2024-12-08T18:40:21,954 Initialized build tracking at C:\Users\ASUS\AppData\Local\Temp\pip-build-tracker-uzy8h0iy
2024-12-08T18:40:21,955 Created build tracker: C:\Users\ASUS\AppData\Local\Temp\pip-build-tracker-uzy8h0iy
2024-12-08T18:40:21,955 Entered build tracker: C:\Users\ASUS\AppData\Local\Temp\pip-build-tracker-uzy8h0iy
2024-12-08T18:40:21,994 Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s
2024-12-08T18:40:21,996 Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-ephem-wheel-cache-upzsdwsg
2024-12-08T18:40:22,014 1 location(s) to search for versions of face-recognition:
2024-12-08T18:40:22,014 * https://pypi.org/simple/face-recognition/
2024-12-08T18:40:22,014 Fetching project page and analyzing links: https://pypi.org/simple/face-recognition/
2024-12-08T18:40:22,014 Getting page https://pypi.org/simple/face-recognition/
2024-12-08T18:40:22,014 Found index url https://pypi.org/simple/
2024-12-08T18:40:22,218 Fetched page https://pypi.org/simple/face-recognition/ as application/vnd.pypi.simple.v1+json
2024-12-08T18:40:22,219   Found link https://files.pythonhosted.org/packages/23/6d/df7c9246e54d9f168948b875d2a3324e3ffc2548166e457570be124147e2/face_recognition-0.1.0.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.0
2024-12-08T18:40:22,219   Found link https://files.pythonhosted.org/packages/1f/23/6407ab68226852172eefd86ee290cf5b4787de9e7b153e19b058a240fe37/face_recognition-0.1.1.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.1
2024-12-08T18:40:22,219   Found link https://files.pythonhosted.org/packages/79/b9/803f3f76407ee94bb8ec9d965fa2837a48926b96b47339d4a4792b417ea7/face_recognition-0.1.4-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.4
2024-12-08T18:40:22,219   Found link https://files.pythonhosted.org/packages/95/a1/58f7904513cb0050226f682a27626ab5bb1cd959d5fa789a10ded150ea69/face_recognition-0.1.4.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.4
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/de/89/6ed56471bb6a5f503230d59be6351c5341b81bd72ade99d8af137e4c0648/face_recognition-0.1.5-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.5
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/92/e0/e9e66c7314f5ba99c1787705ae76361c5ce6029ebe96dab33342f53e8de3/face_recognition-0.1.5.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.5
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/6e/85/a406d6d9e4813f6069de425eca75f5368f0cdd2edd4d65a63a130375d9d2/face_recognition-0.1.6-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.6
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/2d/74/71d8702d5195bcb834800925aef90db1c943b17911e0249192dddf5feac0/face_recognition-0.1.6.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.6
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/e4/46/0ca9468691f8f298430b454bd0cb20d2711911287dd922086603df28ceab/face_recognition-0.1.7-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.7
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/4b/d8/3a03fbe5e684783c2331f107d03c997bd3a1e6f717a8c5ea70c10a88fcc1/face_recognition-0.1.7.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.7
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/93/a6/109a682fcb59fe1529fb87fc9913ff15f8b9917857d1b020834e0f655bc0/face_recognition-0.1.8-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.8
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/10/a8/00d65cc7e76865543cbe7f8bcfed34e49fe9d4a502e443800324696a7cb0/face_recognition-0.1.8.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.8
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/54/bb/1eae78491b1995246247e681c46e87a36f6d3c94f60f1d36be56f9ebb90d/face_recognition-0.1.9-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.9
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/f4/91/0e885ade9413f3ae7243302b1b6ecab6ef59c53d4f35200ae7446ceac737/face_recognition-0.1.9.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.9
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/e6/c2/06a9e4fad7edf961936ff3b673cc1419ded8dabac037d88f953e42c3838d/face_recognition-0.1.10-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.10
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/27/e6/620ce94600e4bc4a1910c98c63b20fb563d1619e7c2f334ee76e84cc7df4/face_recognition-0.1.10.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.10
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/cf/cc/ec839c472a27102b09abc1cace14959d08fccf34544c6924f3f26473cfbb/face_recognition-0.1.11-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.11
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/92/aa/fd00864e0bab93d084879bf149cb25d87850887199f0f1bebd16f926f3dc/face_recognition-0.1.11.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.11
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/58/51/3e9a6dcced830fc33f6680f21b8572211aef9bf224f73ef10a5cb43b598d/face_recognition-0.1.13-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.13
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/38/21/207f900098ef6721903d708de2a2bcf88a17ae44d6386f7506466ee25325/face_recognition-0.1.13.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.13
2024-12-08T18:40:22,220   Found link https://files.pythonhosted.org/packages/72/57/998640d8583b9030d475994e8a5e5aed5c354ad33388bbbf9894cc1914d0/face_recognition-0.1.14-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.1.14
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/38/ba/be95dc2e9f4499a8c569e46e3b94f0c6aac9c8669c7a3761ae02c7740197/face_recognition-0.1.14.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.1.14
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/81/66/53c286bb6b1b69d248732f4a5b1669e1f539fed24f1bd2e75183e0483f8d/face_recognition-0.2.0-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.2.0
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/5c/79/321d02201983630bcdf0feae70ff59ffcfebf1344997211a4d978b159ff6/face_recognition-0.2.0.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.2.0
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/3f/9d/0db113a3b16ff4c6724d8c97844b7083d0b4b1b9d4662ab8f337eb0ba7e0/face_recognition-0.2.1-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.2.1
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/72/ee/e2fed314b72a006d2f5d7b102f845269dc3a3e6cfefb6d43d5342403e336/face_recognition-0.2.1.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.2.1
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/6d/4b/0144e90fa090fd3c381a2e377e25ee0c8c8a1c4ba9250f09cf230a05b6bc/face_recognition-0.2.2-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 0.2.2
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/c2/b4/eef0c099b196c353b9c50cb09131525a60d5b0c05104db678b91ab9f3f98/face_recognition-0.2.2.tar.gz (from https://pypi.org/simple/face-recognition/), version: 0.2.2
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/fa/2d/dbb78acb411de238869451afae8387ae8200d3fe0672f6817a918e03ca49/face_recognition-1.0.0-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 1.0.0
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/03/f9/c025d14d73dc7811ae9f25fa0bfd03323bc6a4023a433f829ce03f1a74d2/face_recognition-1.0.0.tar.gz (from https://pypi.org/simple/face-recognition/), version: 1.0.0
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/75/4f/107fb37a67df87a30947fe5cf40fc7c912104e97a8482cc030829863e061/face_recognition-1.1.0-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 1.1.0
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/7e/01/1c994bcc699cffac95abe641624165f3588c94336ad7548e0e90ed8f6abf/face_recognition-1.1.0.tar.gz (from https://pypi.org/simple/face-recognition/), version: 1.1.0
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/d5/d3/218454c2a08c2602f099f6e1694fff32fbb37e250ad780ae1060fb02b76b/face_recognition-1.2.1-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 1.2.1
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/11/e4/abd2b84f388113f7bcb91bceaf29f4a128a96fa2406b952cf5fb6f2d8c28/face_recognition-1.2.1.tar.gz (from https://pypi.org/simple/face-recognition/), version: 1.2.1
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/28/10/f153bbbc218fc169768aa1c02f2e9178e9241e4af8da56289bdca2c0c217/face_recognition-1.2.2-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 1.2.2
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/e1/da/b607fc6c80eb8e82487b1cb3cff445450d712eb0456f562790c7a761f781/face_recognition-1.2.2.tar.gz (from https://pypi.org/simple/face-recognition/), version: 1.2.2
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/3f/ed/ad9a28042f373d4633fc8b49109b623597d6f193d3bbbef7780a5ee8eef2/face_recognition-1.2.3-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 1.2.3
2024-12-08T18:40:22,221   Found link https://files.pythonhosted.org/packages/a3/f3/99c0fdebe04ab2ed12a6b9a6b6c568a6ae9c757da700215a24aaa8ab9070/face_recognition-1.2.3.tar.gz (from https://pypi.org/simple/face-recognition/), version: 1.2.3
2024-12-08T18:40:22,222   Found link https://files.pythonhosted.org/packages/1e/95/f6c9330f54ab07bfa032bf3715c12455a381083125d8880c43cbe76bb3d0/face_recognition-1.3.0-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/), version: 1.3.0
2024-12-08T18:40:22,222   Found link https://files.pythonhosted.org/packages/6c/49/75dda409b94841f01cbbc34114c9b67ec618265084e4d12d37ab838f4fd3/face_recognition-1.3.0.tar.gz (from https://pypi.org/simple/face-recognition/), version: 1.3.0
2024-12-08T18:40:22,222 Skipping link: not a file: https://pypi.org/simple/face-recognition/
2024-12-08T18:40:22,222 Given no hashes to check 40 links for project 'face-recognition': discarding no candidates
2024-12-08T18:40:22,223 Collecting face_recognition
2024-12-08T18:40:22,224   Obtaining dependency information for face_recognition from https://files.pythonhosted.org/packages/1e/95/f6c9330f54ab07bfa032bf3715c12455a381083125d8880c43cbe76bb3d0/face_recognition-1.3.0-py2.py3-none-any.whl.metadata
2024-12-08T18:40:22,225   Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-unpack-8anyo9dz
2024-12-08T18:40:22,226   Using cached face_recognition-1.3.0-py2.py3-none-any.whl.metadata (21 kB)
2024-12-08T18:40:22,229   Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-metadata-cy3mvovl
2024-12-08T18:40:22,235 1 location(s) to search for versions of face-recognition-models:
2024-12-08T18:40:22,235 * https://pypi.org/simple/face-recognition-models/
2024-12-08T18:40:22,235 Fetching project page and analyzing links: https://pypi.org/simple/face-recognition-models/
2024-12-08T18:40:22,235 Getting page https://pypi.org/simple/face-recognition-models/
2024-12-08T18:40:22,236 Found index url https://pypi.org/simple/
2024-12-08T18:40:22,279 Fetched page https://pypi.org/simple/face-recognition-models/ as application/vnd.pypi.simple.v1+json
2024-12-08T18:40:22,279   Found link https://files.pythonhosted.org/packages/ac/26/5b7745aaa2f2f7a911833cf5d495107e9cdbc81989eb92e34108fa944c36/face_recognition_models-0.1.3-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition-models/), version: 0.1.3
2024-12-08T18:40:22,279   Found link https://files.pythonhosted.org/packages/89/d1/5295e18e6840e38badfeefa27ba0c05f29f72dd357e3a32d96230f4c6448/face_recognition_models-0.1.3.tar.gz (from https://pypi.org/simple/face-recognition-models/), version: 0.1.3
2024-12-08T18:40:22,280   Found link https://files.pythonhosted.org/packages/f3/53/da3a40781cb42164b48e2bab12fc72a3e433a11e08576dcb2f4c452662d9/face_recognition_models-0.2.0.tar.gz (from https://pypi.org/simple/face-recognition-models/), version: 0.2.0
2024-12-08T18:40:22,280   Found link https://files.pythonhosted.org/packages/cf/3b/4fd8c534f6c0d1b80ce0973d01331525538045084c73c153ee6df20224cf/face_recognition_models-0.3.0.tar.gz (from https://pypi.org/simple/face-recognition-models/), version: 0.3.0
2024-12-08T18:40:22,280 Skipping link: not a file: https://pypi.org/simple/face-recognition-models/
2024-12-08T18:40:22,280 Given no hashes to check 1 links for project 'face-recognition-models': discarding no candidates
2024-12-08T18:40:22,282 Using cached wheel link: file:///C:/users/asus/appdata/local/pip/cache/wheels/8f/47/c8/f44c5aebb7507f7c8a2c0bd23151d732d0f0bd6884ad4ac635/face_recognition_models-0.3.0-py2.py3-none-any.whl
2024-12-08T18:40:22,282 Collecting face-recognition-models>=0.3.0 (from face_recognition)
2024-12-08T18:40:22,283   Using cached face_recognition_models-0.3.0-py2.py3-none-any.whl
2024-12-08T18:40:22,325 Requirement already satisfied: Click>=6.0 in c:\users\asus\envhopital\lib\site-packages (from face_recognition) (8.1.7)
2024-12-08T18:40:22,327 1 location(s) to search for versions of dlib:
2024-12-08T18:40:22,327 * https://pypi.org/simple/dlib/
2024-12-08T18:40:22,327 Fetching project page and analyzing links: https://pypi.org/simple/dlib/
2024-12-08T18:40:22,327 Getting page https://pypi.org/simple/dlib/
2024-12-08T18:40:22,327 Found index url https://pypi.org/simple/
2024-12-08T18:40:22,357 Fetched page https://pypi.org/simple/dlib/ as application/vnd.pypi.simple.v1+json
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp27-none-win32) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/76/44/d5ea0c2cfbbce3c12841de8e29ba97e305c585b98c70f8e8f1cb69855f1f/dlib-18.17.100-cp27-none-win32.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp27-none-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/8b/b5/0a1f8e3cc709b845b2fb5592e0576c6194bd735519c779e6a98bf3b0f5c4/dlib-18.17.100-cp27-none-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp34-none-win32) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/9e/b7/5c53db22828707d629e1a94326e14e1bb703b975352e4de427695e9c5d00/dlib-18.17.100-cp34-none-win32.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp34-none-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/66/1f/e1259d07929742e9df98e33bf315f48618809185a8a56a69cc2fd75031a2/dlib-18.17.100-cp34-none-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp35-none-win32) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/e5/41/0818d66f938601f000f28b97967ae6d9f33b43b8b96e092f1873f4a6860b/dlib-18.17.100-cp35-none-win32.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp35-none-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/30/91/ffef19ef1dcbe36fa1fd3d1e01e7cf8177a9edd1a13f51877f8f4623c132/dlib-18.17.100-cp35-none-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Found link https://files.pythonhosted.org/packages/2e/63/801499a0aa588ac29b3400477848caa43e6cbb76af3c1568fcd4b0c21cfd/dlib-18.17.100.zip (from https://pypi.org/simple/dlib/), version: 18.17.100
2024-12-08T18:40:22,360   Found link https://files.pythonhosted.org/packages/98/04/a42d2b901f2c456f86f3c8ace36e3df64d21d7f368b1cac468489205e850/dlib-19.0.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.0.0
2024-12-08T18:40:22,360   Skipping link: none of the wheel's tags (cp35-cp35m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/19/e8/737eae439647f953aa1afe2f25ee88ff3399eb3c936e809f63e197e14036/dlib-19.1.0-cp35-cp35m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,360   Found link https://files.pythonhosted.org/packages/24/e5/af6b0a964ff4b726146128ab4f9cd67293426357c7399effab8240d2b8be/dlib-19.1.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.1.0
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp35-cp35m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/33/e9/a93e58de8dc08c117c37be7598a850e9142dae3c6e385186956a35a621a5/dlib-19.3.1-cp35-cp35m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/f7/3b/0969f13461e6ad84db58a6885c00725a6d97ed8c16b78470d11cc5f1f5de/dlib-19.3.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.3.1
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp35-cp35m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/38/18/92fc25855307bcf582a30034ae657fda205de4f29773323bb388e592f17c/dlib-19.4.0-cp35-cp35m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/1d/76/0f91fde37a83a40cf27e6cabd900b975c2667c522a07ae38e31cda2fa486/dlib-19.4.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.4.0
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/82/68/3c471ddd73afb1378eee507d765bf186e724e8bc81eb44501a99345d89c7/dlib-19.5.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.5.0
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp36-cp36m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/24/ea/81e4fc5b978277899b1c1a63ff358f1f645f9369e59d9b5d9cc1d57c007c/dlib-19.5.1-cp36-cp36m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/47/0c/4cef6abe23eb20ebbc9d7be66a15da21f3997f49a48b415d30a14e5586f0/dlib-19.5.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.5.1
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp36-cp36m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/5b/fa/65bad0783fe624f463ec3c46c0bc062d281ee33c032b3047d99ffcefc9f5/dlib-19.6.0-cp36-cp36m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/3f/2f/2f433c62b56842fc1c3b7855269ca95a6d9757caaa47d6b20f93f13f45c3/dlib-19.6.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.6.0
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp36-cp36m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/3c/97/e8e15bb7288c77085c130f9495f4a884a80a5c9f4634392cdae0aa8f53fe/dlib-19.6.1-cp36-cp36m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/e2/b3/8f689f96b6f2b1f66b6d8d0f91d5b1eb69a7f6a202869e64b0ff6ef74a06/dlib-19.6.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.6.1
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp36-cp36m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/da/06/bd3e241c4eb0a662914b3b4875fc52dd176a9db0d4a2c915ac2ad8800e9e/dlib-19.7.0-cp36-cp36m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/e2/79/6aba1d2b3f9fbcf34d583188d8ff6818952ea875dceedf7c34a869637573/dlib-19.7.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.7.0
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/9b/f4/8d13ff6d9c8520f96760780375af135ae0832325607690d84a367706f1e7/dlib-19.8.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.8.0
2024-12-08T18:40:22,361   Skipping link: none of the wheel's tags (cp36-cp36m-win_amd64) are compatible (run pip debug --verbose to show compatible tags): https://files.pythonhosted.org/packages/0e/ce/f8a3cff33ac03a8219768f0694c5d703c8e037e6aba2e865f9bae22ed63c/dlib-19.8.1-cp36-cp36m-win_amd64.whl (from https://pypi.org/simple/dlib/)
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/eb/99/6e2334b09bf48493b243a0fcb60a60659c847f86cecc1c2a7839609f88eb/dlib-19.8.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.8.1
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/92/7f/7711bf4bb25c7e8ff1c450bf397302d4b75e55abe6de54c75cc05a7b17db/dlib-19.8.2.tar.gz (from https://pypi.org/simple/dlib/), version: 19.8.2
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/23/7a/891d4c46cdf755fb69eec0f06a53033d3754bf68ace5342ad56aeb540aa1/dlib-19.9.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.9.0
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/54/ff/5781a11bdc0d1c90a813f3f66d460abf628e5114856a842d18d43f709a7a/dlib-19.10.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.10.0
2024-12-08T18:40:22,361   Found link https://files.pythonhosted.org/packages/f2/02/dbffb7023494e1e39981ca65010b27501e844a0c8e79e1a3034ad8bf3734/dlib-19.12.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.12.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/79/26/8ef2835606b4b37f704862b2db55edb89675daca0b82718244fca718cddd/dlib-19.13.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.13.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/d5/5c/aa64510aa354d562ecba7edecd500b020280741140e5d91ff5ec1c6f8289/dlib-19.13.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.13.1
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/97/08/0d2b53b3845d562d01108e482ae45596a0d13bf58c63172df954ff0b53ef/dlib-19.14.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.14.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/df/aa/6a9bb2a763107bb2606d6ee1aa65fcd3b51375a9ef6436e9c9280b0dd63c/dlib-19.15.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.15.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/35/8d/e4ddf60452e2fb1ce3164f774e68968b3f110f1cb4cd353235d56875799e/dlib-19.16.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.16.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/05/57/e8a8caa3c89a27f80bc78da39c423e2553f482a3705adc619176a3a24b36/dlib-19.17.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.17.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/1e/62/aacb236d21fbd08148b1d517d58a9d80ea31bdcd386d26f21f8b23b1eb28/dlib-19.18.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.18.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/63/92/05c3b98636661cb80d190a5a777dd94effcc14c0f6893222e5ca81e74fbc/dlib-19.19.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.19.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/f1/40/e407366816646bb452578997e6ca2140f004eec8f611e8512ee429ded2be/dlib-19.20.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.20.0
2024-12-08T18:40:22,362   Found link https://files.pythonhosted.org/packages/a4/7b/2f7f29f460629a8143b2deea1911e2fb1d9d88d29bf645ba321461588e88/dlib-19.21.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.21.0
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/99/2c/ef681c1c717ace11040f9e99fe22dafc843cdd6085eb6120e7ab2a5c662b/dlib-19.21.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.21.1
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/11/93/ec41d6ef7e769977aa08e49441c52276da27859f12dcbf1c6deb96ce5e9f/dlib-19.22.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.22.0
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/f0/a2/ba6163c09fb427990180afd8d625bcecc5555af699c253193c35ffd48c4f/dlib-19.22.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.22.1
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/25/0f/05e70b2ebf6a539c4c397c7c3dd19b69e0b3d650443e12a8af4866f65b03/dlib-19.23.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.23.0
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/f9/0f/74ba48d5136741feff3ceba90815034d34e10902cd16b44ae170226ff43e/dlib-19.23.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.23.1
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/e5/3b/7a8522a5c2ef6ff6252e46b0788b3d2c2280198c49d6ecf3b576b171045f/dlib-19.24.0.tar.gz (from https://pypi.org/simple/dlib/), version: 19.24.0
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/ec/e3/4fed0fb511a88a46365c2da493b73081a5e1a21960bdb18089b27cbc768b/dlib-19.24.1.tar.gz (from https://pypi.org/simple/dlib/), version: 19.24.1
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/2e/9d/4e1003c044bdb07f7c0d83ae87d694e10e5a6c296b84566aa9a6ec9eed2a/dlib-19.24.2.tar.gz (from https://pypi.org/simple/dlib/), version: 19.24.2
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/af/a4/226dbb659e913a4a149b35980e87e10050ea39a0dceac934e9e73cccbef1/dlib-19.24.4.tar.gz (from https://pypi.org/simple/dlib/), version: 19.24.4
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/8d/74/c028536e66f63f689bd76a58a756a184fc2feb3a35bb877a92677df778dc/dlib-19.24.5.tar.gz (from https://pypi.org/simple/dlib/), version: 19.24.5
2024-12-08T18:40:22,363   Found link https://files.pythonhosted.org/packages/3c/07/a2693a94ac678c442c4cfe269d24f63053b14410ef2d09957a762eeb4b8e/dlib-19.24.6.tar.gz (from https://pypi.org/simple/dlib/), version: 19.24.6
2024-12-08T18:40:22,363 Skipping link: not a file: https://pypi.org/simple/dlib/
2024-12-08T18:40:22,366 Given no hashes to check 28 links for project 'dlib': discarding no candidates
2024-12-08T18:40:22,368 Collecting dlib>=19.7 (from face_recognition)
2024-12-08T18:40:22,370   Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-unpack-xq8241x7
2024-12-08T18:40:22,372   Using cached dlib-19.24.6.tar.gz (3.4 MB)
2024-12-08T18:40:25,110   Added dlib>=19.7 from https://files.pythonhosted.org/packages/3c/07/a2693a94ac678c442c4cfe269d24f63053b14410ef2d09957a762eeb4b8e/dlib-19.24.6.tar.gz (from face_recognition) to build tracker 'C:\\Users\\ASUS\\AppData\\Local\\Temp\\pip-build-tracker-uzy8h0iy'
2024-12-08T18:40:25,111   Running setup.py (path:C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py) egg_info for package dlib
2024-12-08T18:40:25,112   Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel
2024-12-08T18:40:25,112   Running command python setup.py egg_info
2024-12-08T18:40:25,746   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:234: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:25,747     major = re.findall("set\(CPACK_PACKAGE_VERSION_MAJOR.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:25,747   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:235: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:25,747     minor = re.findall("set\(CPACK_PACKAGE_VERSION_MINOR.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:25,747   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:236: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:25,747     patch = re.findall("set\(CPACK_PACKAGE_VERSION_PATCH.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:25,859   running egg_info
2024-12-08T18:40:25,871   creating C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info
2024-12-08T18:40:25,871   writing C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info\PKG-INFO
2024-12-08T18:40:25,913   writing dependency_links to C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info\dependency_links.txt
2024-12-08T18:40:25,915   writing top-level names to C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info\top_level.txt
2024-12-08T18:40:25,918   writing manifest file 'C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info\SOURCES.txt'
2024-12-08T18:40:25,973   reading manifest file 'C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info\SOURCES.txt'
2024-12-08T18:40:25,974   reading manifest template 'MANIFEST.in'
2024-12-08T18:40:26,983   no previously-included directories found matching 'tools\python\build*'
2024-12-08T18:40:26,985   no previously-included directories found matching 'dlib\cmake_utils\*\build*'
2024-12-08T18:40:26,987   no previously-included directories found matching 'dlib\test'
2024-12-08T18:40:26,990   warning: no previously-included files matching '*.pyc' found anywhere in distribution
2024-12-08T18:40:27,132   writing manifest file 'C:\Users\ASUS\AppData\Local\Temp\pip-pip-egg-info-5s7zlqel\dlib.egg-info\SOURCES.txt'
2024-12-08T18:40:27,179   Source in c:\users\asus\appdata\local\temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed has version 19.24.6, which satisfies requirement dlib>=19.7 from https://files.pythonhosted.org/packages/3c/07/a2693a94ac678c442c4cfe269d24f63053b14410ef2d09957a762eeb4b8e/dlib-19.24.6.tar.gz (from face_recognition)
2024-12-08T18:40:27,180   Removed dlib>=19.7 from https://files.pythonhosted.org/packages/3c/07/a2693a94ac678c442c4cfe269d24f63053b14410ef2d09957a762eeb4b8e/dlib-19.24.6.tar.gz (from face_recognition) from build tracker 'C:\\Users\\ASUS\\AppData\\Local\\Temp\\pip-build-tracker-uzy8h0iy'
2024-12-08T18:40:27,182 Requirement already satisfied: numpy in c:\users\asus\envhopital\lib\site-packages (from face_recognition) (2.0.2)
2024-12-08T18:40:27,183 Requirement already satisfied: Pillow in c:\users\asus\envhopital\lib\site-packages (from face_recognition) (10.4.0)
2024-12-08T18:40:27,187 Requirement already satisfied: colorama in c:\users\asus\envhopital\lib\site-packages (from Click>=6.0->face_recognition) (0.4.6)
2024-12-08T18:40:27,201 Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-unpack-jo1gghdp
2024-12-08T18:40:27,202 Using cached face_recognition-1.3.0-py2.py3-none-any.whl (15 kB)
2024-12-08T18:40:27,243 Downloading link https://files.pythonhosted.org/packages/1e/95/f6c9330f54ab07bfa032bf3715c12455a381083125d8880c43cbe76bb3d0/face_recognition-1.3.0-py2.py3-none-any.whl (from https://pypi.org/simple/face-recognition/) to C:\Users\ASUS\AppData\Local\Temp\pip-unpack-jo1gghdp\face_recognition-1.3.0-py2.py3-none-any.whl
2024-12-08T18:40:27,245 Building wheels for collected packages: dlib
2024-12-08T18:40:27,247   Created temporary directory: C:\Users\ASUS\AppData\Local\Temp\pip-wheel-or0x1l5q
2024-12-08T18:40:27,248   Destination directory: C:\Users\ASUS\AppData\Local\Temp\pip-wheel-or0x1l5q
2024-12-08T18:40:27,248   Running command python setup.py bdist_wheel
2024-12-08T18:40:27,791   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:234: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:27,791     major = re.findall("set\(CPACK_PACKAGE_VERSION_MAJOR.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:27,791   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:235: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:27,791     minor = re.findall("set\(CPACK_PACKAGE_VERSION_MINOR.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:27,791   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:236: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:27,791     patch = re.findall("set\(CPACK_PACKAGE_VERSION_PATCH.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:27,855   running bdist_wheel
2024-12-08T18:40:27,887   running build
2024-12-08T18:40:27,888   running build_ext

2024-12-08T18:40:27,889   ================================================================================
2024-12-08T18:40:27,889   ================================================================================
2024-12-08T18:40:27,889   ================================================================================

2024-12-08T18:40:27,889                      CMake is not installed on your system!

2024-12-08T18:40:27,889       Or it is possible some broken copy of cmake is installed on your system.
2024-12-08T18:40:27,889       It is unfortunately very common for python package managers to include
2024-12-08T18:40:27,889       broken copies of cmake.  So if the error above this refers to some file
2024-12-08T18:40:27,889       path to a cmake file inside a python or anaconda or miniconda path then you
2024-12-08T18:40:27,889       should delete that broken copy of cmake from your computer.

2024-12-08T18:40:27,889       Instead, please get an official copy of cmake from one of these known good
2024-12-08T18:40:27,889       sources of an official cmake:
2024-12-08T18:40:27,889           - cmake.org (this is how windows users should get cmake)
2024-12-08T18:40:27,890           - apt install cmake (for Ubuntu or Debian based systems)
2024-12-08T18:40:27,890           - yum install cmake (for Redhat or CenOS based systems)

2024-12-08T18:40:27,890       On a linux machine you can run `which cmake` to see what cmake you are
2024-12-08T18:40:27,890       actually using.  If it tells you it's some cmake from any kind of python
2024-12-08T18:40:27,890       packager delete it and install an official cmake.

2024-12-08T18:40:27,890       More generally, cmake is not installed if when you open a terminal window
2024-12-08T18:40:27,890       and type
2024-12-08T18:40:27,890          cmake --version
2024-12-08T18:40:27,890       you get an error.  So you can use that as a very basic test to see if you
2024-12-08T18:40:27,890       have cmake installed.  That is, if cmake --version doesn't run from the
2024-12-08T18:40:27,890       same terminal window from which you are reading this error message, then
2024-12-08T18:40:27,890       you have not installed cmake.  Windows users should take note that they
2024-12-08T18:40:27,890       need to tell the cmake installer to add cmake to their PATH.  Since you
2024-12-08T18:40:27,890       can't run commands that are not in your PATH.  This is how the PATH works
2024-12-08T18:40:27,890       on Linux as well, but failing to add cmake to the PATH is a particularly
2024-12-08T18:40:27,890       common problem on windows and rarely a problem on Linux.

2024-12-08T18:40:27,891   ================================================================================
2024-12-08T18:40:27,891   ================================================================================
2024-12-08T18:40:27,891   ================================================================================
2024-12-08T18:40:27,916   ERROR: python setup.py bdist_wheel exited with 1
2024-12-08T18:40:27,921   [bold magenta]full command[/]: [blue]'C:\Users\ASUS\envhopital\Scripts\python.exe' -u -c '
2024-12-08T18:40:27,921   exec(compile('"'"''"'"''"'"'
2024-12-08T18:40:27,921   # This is <pip-setuptools-caller> -- a caller that pip uses to run setup.py
2024-12-08T18:40:27,921   #
2024-12-08T18:40:27,921   # - It imports setuptools before invoking setup.py, to enable projects that directly
2024-12-08T18:40:27,921   #   import from `distutils.core` to work with newer packaging standards.
2024-12-08T18:40:27,921   # - It provides a clear error message when setuptools is not installed.
2024-12-08T18:40:27,921   # - It sets `sys.argv[0]` to the underlying `setup.py`, when invoking `setup.py` so
2024-12-08T18:40:27,921   #   setuptools doesn'"'"'t think the script is `-c`. This avoids the following warning:
2024-12-08T18:40:27,921   #     manifest_maker: standard file '"'"'-c'"'"' not found".
2024-12-08T18:40:27,921   # - It generates a shim setup.py, for handling setup.cfg-only projects.
2024-12-08T18:40:27,921   import os, sys, tokenize
2024-12-08T18:40:27,921   
2024-12-08T18:40:27,921   try:
2024-12-08T18:40:27,921       import setuptools
2024-12-08T18:40:27,921   except ImportError as error:
2024-12-08T18:40:27,921       print(
2024-12-08T18:40:27,921           "ERROR: Can not execute `setup.py` since setuptools is not available in "
2024-12-08T18:40:27,921           "the build environment.",
2024-12-08T18:40:27,921           file=sys.stderr,
2024-12-08T18:40:27,921       )
2024-12-08T18:40:27,921       sys.exit(1)
2024-12-08T18:40:27,921   
2024-12-08T18:40:27,921   __file__ = %r
2024-12-08T18:40:27,921   sys.argv[0] = __file__
2024-12-08T18:40:27,921   
2024-12-08T18:40:27,921   if os.path.exists(__file__):
2024-12-08T18:40:27,921       filename = __file__
2024-12-08T18:40:27,921       with tokenize.open(__file__) as f:
2024-12-08T18:40:27,921           setup_py_code = f.read()
2024-12-08T18:40:27,921   else:
2024-12-08T18:40:27,921       filename = "<auto-generated setuptools caller>"
2024-12-08T18:40:27,921       setup_py_code = "from setuptools import setup; setup()"
2024-12-08T18:40:27,921   
2024-12-08T18:40:27,921   exec(compile(setup_py_code, filename, "exec"))
2024-12-08T18:40:27,921   '"'"''"'"''"'"' % ('"'"'C:\\Users\\ASUS\\AppData\\Local\\Temp\\pip-install-tjp8xu4s\\dlib_3a01565f5fae46bbbb5319e2a4e015ed\\setup.py'"'"',), "<pip-setuptools-caller>", "exec"))' bdist_wheel -d 'C:\Users\ASUS\AppData\Local\Temp\pip-wheel-or0x1l5q'[/]
2024-12-08T18:40:27,921   [bold magenta]cwd[/]: C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\\
2024-12-08T18:40:27,921   ERROR: Failed building wheel for dlib
2024-12-08T18:40:27,922   Running setup.py clean for dlib
2024-12-08T18:40:27,923   Running command python setup.py clean
2024-12-08T18:40:28,391   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:234: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:28,391     major = re.findall("set\(CPACK_PACKAGE_VERSION_MAJOR.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:28,391   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:235: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:28,391     minor = re.findall("set\(CPACK_PACKAGE_VERSION_MINOR.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:28,391   C:\Users\ASUS\AppData\Local\Temp\pip-install-tjp8xu4s\dlib_3a01565f5fae46bbbb5319e2a4e015ed\setup.py:236: SyntaxWarning: invalid escape sequence '\('
2024-12-08T18:40:28,391     patch = re.findall("set\(CPACK_PACKAGE_VERSION_PATCH.*\"(.*)\"", open(cmake_file).read())[0]
2024-12-08T18:40:28,451   running clean
2024-12-08T18:40:28,468   'build\lib.win-amd64-cpython-312' does not exist -- can't clean it
2024-12-08T18:40:28,468   'build\bdist.win-amd64' does not exist -- can't clean it
2024-12-08T18:40:28,468   'build\scripts-3.12' does not exist -- can't clean it
2024-12-08T18:40:28,490 Failed to build dlib
2024-12-08T18:40:28,674 Remote version of pip: 24.3.1
2024-12-08T18:40:28,674 Local version of pip:  24.3.1
2024-12-08T18:40:28,677 Was pip installed by pip? True
2024-12-08T18:40:28,678 ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (dlib)
2024-12-08T18:40:28,679 Exception information:
2024-12-08T18:40:28,679 Traceback (most recent call last):
2024-12-08T18:40:28,679   File "C:\Users\ASUS\envhopital\Lib\site-packages\pip\_internal\cli\base_command.py", line 105, in _run_wrapper
2024-12-08T18:40:28,679     status = _inner_run()
2024-12-08T18:40:28,679              ^^^^^^^^^^^^
2024-12-08T18:40:28,679   File "C:\Users\ASUS\envhopital\Lib\site-packages\pip\_internal\cli\base_command.py", line 96, in _inner_run
2024-12-08T18:40:28,679     return self.run(options, args)
2024-12-08T18:40:28,679            ^^^^^^^^^^^^^^^^^^^^^^^
2024-12-08T18:40:28,679   File "C:\Users\ASUS\envhopital\Lib\site-packages\pip\_internal\cli\req_command.py", line 67, in wrapper
2024-12-08T18:40:28,679     return func(self, options, args)
2024-12-08T18:40:28,679            ^^^^^^^^^^^^^^^^^^^^^^^^^
2024-12-08T18:40:28,679   File "C:\Users\ASUS\envhopital\Lib\site-packages\pip\_internal\commands\install.py", line 434, in run
2024-12-08T18:40:28,679     raise InstallationError(
2024-12-08T18:40:28,679 pip._internal.exceptions.InstallationError: ERROR: Failed to build installable wheels for some pyproject.toml based projects (dlib)
2024-12-08T18:40:28,695 Removed build tracker: 'C:\\Users\\ASUS\\AppData\\Local\\Temp\\pip-build-tracker-uzy8h0iy'
